layouts/partials/fragments/contact.html

```diff
@@ -53,7 +57,7 @@ <h2>{{- . | markdownify -}}</h2>
                       type="text"
                       data-validation
                       data-validation-error-msg="{{ .error | default (i18n "contact.defaultNameError") }}"
-                      placeholder="{{ with .button_text }}{{ . | markdownify }}{{ end }}"
```

Comment:
Did we decide with "text" again? thought we had it changed.

Reply:
No afaik we decided on .button_text. Not sure what happened here. Maybe just some rebase hiccup. @mpourismaiel

Reply:
Because this file contained `text` and I thought that was correct one. I'll change it ASAP.
https://github.com/mpourismaiel/syna/blob/d9b291ee3028849a7bcc7fd3e3d2ed78d688a186/exampleSite/content/_index/contact.md

Reply:
Actually I think `text` is more accurate in this case since `buttent_text` is not descriptive for the input's placeholder.

Reply:
Ahh you are correct. The placeholder is actually .text.

.button_text was just the button. Totally missed that.